rust_src/remacs-lib/docfile.rs

```diff
@@ -137,6 +138,24 @@ pub extern "C" fn scan_rust_file(
             let name = CString::new(&caps[1]).unwrap();
             let value = CString::new(&caps[2]).unwrap();
             add_global(SYMBOL, name.as_ptr(), 0, value.as_ptr());
+        } else if line.starts_with("defvar_") {
+            // defvar_lisp!(f_Vpost_self_insert_hook, "post-self-insert-hook", Qnil);
+            lazy_static! {
+                static ref RE: Regex = Regex::new(r#"defvar_(lisp)!\((.+?),\s+"(.+?)",\s+(.+?)\);"#).unwrap();
+            }
+            let caps = RE.captures(line).unwrap();
+            let kindstr = &caps[1];
+            let kind = match kindstr {
+                "lisp" => LISP_OBJECT,
+                "lisp_nopro" => LISP_OBJECT,
+                "bool" => BOOLEAN,
+                "int" => EMACS_INTEGER,
+                _ => panic!("oops"),
```

Comment:
Could we say why we're panicking in the message here? It helps future folks if they hit it :)

Reply:
I knew I forgot something